src/tokens-studio/spectrum2-colors/spectrum2/icon/light.json

```diff
@@ -1,16 +1,628 @@
+              "name": "icon-color-seafoam-primary-hover"
+            }
+          }
+        },
+        "down": {
+          "value": "{Palette.seafoam.1000}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "",
+              "name": "icon-color-seafoam-primary-down"
+            }
+          }
+        }
+      },
+      "silver": {
+        "default": {
+          "value": "{Palette.silver.700}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "",
+              "name": "icon-color-silver-primary-default"
+            }
+          }
+        },
+        "hover": {
+          "value": "{Palette.silver.800}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "",
+              "name": "icon-color-silver-primary-hover"
+            }
+          }
+        },
+        "down": {
+          "value": "{Palette.silver.900}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "",
+              "name": "icon-color-silver-primary-down"
+            }
+          }
+        }
+      },
+      "turquoise": {
+        "default": {
+          "value": "{Palette.turquoise.700}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "",
+              "name": "icon-color-turquoise-primary-default"
+            }
+          }
+        },
+        "hover": {
+          "value": "{Palette.turquoise.800}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "",
+              "name": "icon-color-turquoise-primary-hover"
+            }
+          }
+        },
+        "down": {
+          "value": "{Palette.turquoise.900}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "",
+              "name": "icon-color-turquoise-primary-down"
+            }
+          }
+        }
+      },
+      "yellow": {
+        "default": {
+          "value": "{Palette.yellow.400}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "59cd6057-b3d8-4bdf-b752-7df17c2c4a95",
+              "name": "icon-color-yellow-primary-default"
+            }
+          }
+        },
+        "hover": {
+          "value": "{Palette.yellow.500}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "",
+              "name": "icon-color-yellow-primary-hover"
+            }
+          }
+        },
+        "down": {
+          "value": "{Palette.yellow.600}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "",
+              "name": "icon-color-yellow-primary-hover"
```

Comment:
@nabuhasan this name doesn't match. Is this `icon-color-yellow-primary-down`?

Reply:
Fixed!